src/lib/renderer/Tags.svelte

```diff
@@ -0,0 +1,20 @@
+<script>
```

Comment:
Not sure if we need a separate component, could just be in `Markdoc.svelte`, thoughts?

Reply:
Factoring this out means that we only run parse and transform the doc once. Otherwise we'll do so for each child in the tree, no?

Reply:
Thanks for clarifying, I missed that it was using `<svelte:self>`, my bad. It def. needs to be in a different component.